backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -165,70 +166,74 @@ export class AdminResolver {
 
   @Authorized([RIGHTS.CREATE_PENDING_CREATION])
   @Mutation(() => [Number])
-  async createPendingCreation(
-    @Args() { email, amount, memo, creationDate }: CreatePendingCreationArgs,
+  async adminCreateContribution(
+    @Args() { email, amount, memo, creationDate }: AdminCreateContributionArgs,
     @Ctx() context: Context,
   ): Promise<Decimal[]> {
+    logger.trace('adminCreateContribution...')
     const user = await dbUser.findOne({ email }, { withDeleted: true })
     if (!user) {
       throw new Error(`Could not find user with email: ${email}`)
     }
     if (user.deletedAt) {
-      throw new Error('This user was deleted. Cannot make a creation.')
+      throw new Error('This user was deleted. Cannot create a contribution.')
     }
     if (!user.emailChecked) {
-      throw new Error('Creation could not be saved, Email is not activated')
+      throw new Error('Contribution could not be saved, Email is not activated')
     }
     const moderator = getUser(context)
+    logger.trace('moderator: ', moderator.id)
     const creations = await getUserCreation(user.id)
+    logger.trace('creations', creations)
     const creationDateObj = new Date(creationDate)
     if (isCreationValid(creations, amount, creationDateObj)) {
```

Comment:
- rename isCreationValid() to isContributionValid()
- rename the parameter creations to contributions or contributionList
- rename the creationDate to createdAt, because its confusing to distingue between creation/contribution-Obj and creationDate

Reply:
creations are actually creation transactions, the date should have another name. It corresponds to the month the creation is for, stored in `contributionDate`